docs/usage/configuration-options.md

```diff
@@ -466,6 +466,18 @@ If configured to `true`, then dependency extraction will be done using the relev
 
 This option applies only to the `gradle` manager.
 
+## defaultRegistryUrls
+
+This configuration option enables overriding of a datasource's default registries.
+Like `registryUrls`, it is only applicable when the datasource's `customRegistrySupport` value is set to `true`.
+
+Default registries are only used when both:
+
+- The manager does not extract any `registryUrls` values, and
+- No `registryUrls` values have been applied via config, such as `packageRules`
+
+In other words, you can think of it as a way to specify the "fallback" registries for a datasource, where supported.
```

Comment:
```suggestion
Think of `defaultRegistryUrls` as a way to specify the "fallback" registries for a datasource, where supported.
```

We can strip the `In other words,` part.

I think `it` refers to the `defaultRegistryUrls` config option?